specification/keyvault/data-plane/Security.KeyVault.SecurityDomain/routes.tsp

```diff
@@ -26,6 +26,7 @@ interface HsmSecurityDomain {
    */
   #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation necessary for Key Vault"
   @pollingOperation(HsmSecurityDomain.downloadPending)
+  @finalOperation(HsmSecurityDomain.downloadPending)
```

Comment:
Seems the operation ID should be case-sensitive, so we should either call the interface `HSMSecurityDomain` or use a decorator to do so, if possible.